src/main/webapp/components/InputField.js

```diff
@@ -12,7 +12,7 @@ export const InputField = hh(class InputField extends Component {
   render() {
 
     return (
-      div({ className: "inputField " + (this.props.error === true ? 'inputFieldError' : '') }, [
+      div({ className: "inputField " + (this.props.error === true ? 'inputFieldError' : this.props.readOnly ? 'inputFieldReadOnly' : '') }, [

```

Comment:
this implies precedence of this.props.error over this.props.readOnly. It is the desired behavior?

Reply:
Yes.